src/gstvimbasrc.c

```diff
@@ -377,21 +377,21 @@ static void gst_vimbasrc_class_init(GstVimbaSrcClass *klass)
         g_param_spec_int(
             "offsetx",
             "OffsetX feature setting",
-            "Horizontal offset from the origin to the region of interest (in pixels).",
+            "Horizontal offset from the origin to the region of interest (in pixels). If no explicit value is passed the RoI will be centered in the sensor.",
             0,
             G_MAXINT,
-            0,
+            G_MAXINT,
```

Comment:
Reset default value to 0 so that centering is not enabled by default. This keeps more it more consistent with general camera API

```suggestion
            0,
```